src/tools/other/pb2nc/pb2nc.cc

```diff
@@ -1345,17 +1345,21 @@ void process_pbfile(int i_pb) {
          }
       }
 
+      // Initialize for CAPE and PBL
       if (cal_cape || cal_mlcape) {
          cape_level = 0;
+         cape_qm = bad_data_float;
       }
 
       do_pbl = cal_pbl && 0 == strcmp("ADPUPA", hdr_typ);
       if (do_pbl) {
          pbl_level = 0;
+         pbl_qm = bad_data_float;
       }
```

Comment:
@hsoh-u I'm not totally confident in these lines numbers and changes. Rather than cause a bigger mess now, and you needing to clean it up later, I think it'd be easier for you to do this directly.

Can you please just check out the current state of the feature_2278_qm_docs branch, make these changes you recommend, commit them, and push them back up to GitHub?